pkg/3scale/amp/component/backend.go

```diff
@@ -394,6 +405,10 @@ func (backend *Backend) buildBackendWorkerEnv() []v1.EnvVar {
 	result = append(result,
 		helper.EnvVarFromSecret("CONFIG_EVENTS_HOOK", "system-events-hook", "URL"),
 		helper.EnvVarFromSecret("CONFIG_EVENTS_HOOK_SHARED_SECRET", "system-events-hook", "PASSWORD"),
+		// TODO: Add as EnvVar or reference to CM?
+		// TODO: Enable metrics endpoint only when monitoring is enabled by CR? Other components have monitoring enabled no matter monitoring field is enabled or not.
+		v1.EnvVar{Name: "CONFIG_WORKER_PROMETHEUS_METRICS_PORT", Value: BackendWorkerMetricsPortStr},
```

Comment:
helper.EnvVarFromValue can be used for this

Reply:
It is already one line thing. I prefer like this because it is already clean and clear.
```
func EnvVarFromValue(name string, value string) v1.EnvVar {    
    return v1.EnvVar{                                          
        Name:  name,                                           
        Value: value,                                          
    }                                                          
}                                                              
```

Reply:
:+1:  as you prefer